### ISSUE TITLE
[Bug] Mission editor in custom maps does not save Wave Cap

### ISSUE THREAD

**Issue by Mylifefighter** [KnuckleCracker/CW4-bug-tracker#522]
Very simple, the mission editor will not save the "wave cap" parameter when editing custom maps. 

How to recreate the bug:
->Make a new custom map (16x16)
->Go to Editor->Game->Wave Cap
->Change it to anything besides 35 (like 5)
->Apply the changes
->Save the map
->Load the map
-> _The wave cap parameter is now 35 again._

--Alternatively--
(Same as above)
->Save the map
->Exit the mission
->Edit the map
-> _The wave cap parameter is now 35 again._

Log File: [output_log.txt](https://github.com/KnuckleCracker/CW4-bug-tracker/files/5722040/output_log.txt)
Sample Map from above: [Sample Map.zip](https://github.com/KnuckleCracker/CW4-bug-tracker/files/5722041/Sample.Map.zip)
Photos of the bug: Before loading
![MissionEditorPreLoad](https://user-images.githubusercontent.com/20076480/102733820-1adb5500-42f3-11eb-99c7-17cabe88b082.png)

After loading
![MissionEditorPostLoad](https://user-images.githubusercontent.com/20076480/102733818-19aa2800-42f3-11eb-8a26-d3fed21452bb.png)

All other game parameters appear to be saved & loaded properly.

**Comment by knucracker** [comment]
Same as #414.   Should be fixed in the 1.1.0 build that is currently available on steam in the betas tab for Creeper World 4. This will be rolled out  soon as the default version.

**Comment by Karsten75** [comment]
For note:
If you would like to try the 1.1.0 version of CW4, it is now available on the betas tab within steam. It is not the default, so you have to switch to it if you want to try it. I'm doing it this way so any brave soul can try it out before it becomes the default version later this week (and the entire creeper nation gets it whether they want it or not :slight_smile: ).  In steam, right click the game and select properties. Make your way to the betas tab. In there will be 1.1.0 in the dropdown. If not, restart the steam client. The following are the changes:

```
- Updated to Unity 2019.4

- Colonies remembers last selected map when returning to the Colonies screen.

- ERNs no longer move over creeper, they move through it.

- Eggs show yellow smoke when they are popped over shields and do not deposit all of their payload.

- MicroRifts can be placed on platforms.

- AC Bombers get a large speed increase when ERN'd.

- Terps get a speed increase when ERN'd.

- Greenar Refineris get a 2x efficiency boost when ERN'd.

- Game message sizing support for large, small and very wide displays.

- Added support for full screen exclusive mode. The default is still windowed full screen.

- Allow binding key controls for different simulatneous actions (you can set the same key to 2 different things).

- Allow the use of additional mouse buttons in the controls settings.

- Shields show a range before they are built and when being moved.

- Added an option to turn off "Auto Cam". When deselected, the pitch of the camera no longer changes when changing altitude. It is set by rotating the camera, which allows right/left/up/down when auto cam is turned off.

- Added global sound mute button on the main menu. Same setting as found in the game near the top right.

- Various Mission fixes and small updates

- Numerous bug fixes
```
